react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      display: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.showMenu = this.showMenu.bind(this);
+    this.hideMenu = this.hideMenu.bind(this);
+  }
+
+  showMenu() {
+    document.querySelector('.menu').style.width = '20%';
```

Comment:
We don't want to manipulate the DOM instead, pass the appetite props to your react elements in your render method.

Reply:
Gotcha, I've removed the elements that manipulate the DOM and added some modifier classes to pass on new style properties on the condition that this.state.display is true